<launch>
  <arg name="LAUNCH_DYNAMIC_TF_PUBLISHER" default="true" />
  <arg name="models" default="$(find jsk_interactive_marker)/launch/models/set_urdf.yaml" />
  <node pkg="jsk_interactive_marker" type="urdf_model_marker" name="set_urdf_marker" clear_params="true" output="screen" >
    <rosparam command="load" param="model_config" file="$(arg models)"/>
    <param name="use_dynamic_tf" value="$(arg LAUNCH_DYNAMIC_TF_PUBLISHER)"/>
    <remap from="/tf" to="/tf_low_frequency"/>
  </node>

  <group if="$(arg LAUNCH_DYNAMIC_TF_PUBLISHER)">
    <node pkg="dynamic_tf_publisher" type="tf_publish.py" name="dynamic_tf_publisher_for_set_urdf_marker">
      <param name="use_cache" type="bool" value="false"/>
      <remap from="/tf" to="/tf_low_frequency"/>
    </node>

  </group>
</launch>
